tox.ini

```diff
@@ -67,24 +67,16 @@ commands =
   coverage3 report
 
 [testenv:docs]
-basepython = python3
+# Editable mode breaks macOS: https://github.com/sphinx-doc/sphinx/issues/10943
+usedevelop = False
+basepython = python3.9
```

Comment:
I'm ok with setting `basepython` because I have many pythons on my default path, but I don't think that's standard on Macs, so it might be a bit of a nuisance.  There shouldn't be anything inherent that prevents the build from working on any Python 3, it's just that we happen to use 3.9 because at the time I last updated the scripts, I think 3.10 was relatively new and not all our dependencies had released for it, so 3.9 was the newest practical one.

Reply:
oh, actually this got removed while I was still looking at the diff lol

Reply:
Yeah, this wasn't intentional. I manually set `python3.9` because otherwise it uses Python 3.11, which still has some issues for me with Sphinx even after this PR.

I don't think we should set basepython because we force all users to have that same binary on their machine. I only forgot to remove this before committing - my bad